### PR TITLE
feat: include lobbyToken in GET lobby/:id response

### PR DESCRIPTION
**Pull request by darren2hang**
Small change: The lobbyToken of the lobby is included in the response to lobby/:id GET

### CODE REVIEW

File: www/lobby/id.ts [hunk 1]
```diff
@@ -56,7 +56,8 @@ lobby_id_router.get('/:id', async (req: Request, res: Response) => {
   const { lobby } = verified;
   if (!lobby.participants.includes(userId)) return res.status(406).json('User is not part of the lobby').end();
 
-  res.status(200).json(lobby.getClientResponse());
+  const lobbyToken = genJwt(lobbyId, EXPIRATION.SEVEN_DAYS);
+  res.status(200).json({...lobby.getClientResponse(), lobbyToken: lobbyToken});
```
Comment (BryanPan342) [inline www/lobby/id.ts:60]: only if the user is the manager!

File: www/lobby/id.ts [hunk 2]
```diff
@@ -56,7 +56,14 @@ lobby_id_router.get('/:id', async (req: Request, res: Response) => {
...
 
-  res.status(200).json(lobby.getClientResponse());
+  let resObj: Record<string, unknown> = lobby.getClientResponse();
+
+  if (userId == lobby.managerId) {
+    const lobbyToken = genJwt(lobbyId, EXPIRATION.SEVEN_DAYS);
+    resObj = {...resObj, lobbyToken: lobbyToken};
+  }
+
+  res.status(200).json(resObj);
```
Comment (BryanPan342) [inline www/lobby/id.ts:66]:
```suggestion
  res.status(200).json(userId === lobby.managerId
    ? lobby.getClientResponse()
    : {...lobby.getClientResponse, lobbyToken: genJwt(lobbyId, EXPIRATION.ONE_HOUR}
```

**Review by BryanPan342 — CHANGES_REQUESTED** [review]
me thinks this is cleaner

**Review by BryanPan342 — DISMISSED** [review]
LGTM

**Review by trevorong — APPROVED** [review]
lgtm

**Pull request merged**